packages/web/src/components/spa/Footer/Footer.jsx

```diff
@@ -12,30 +12,37 @@ export const Footer = ({ colour, iconcolor }) => {
             </div>
             <div className="social-media">
                 <div className="links">
-                    <a
-                        href="https://www.facebook.com/TheHubAUBG"
-                        target="_blank"
-                    >
-                        <FaFacebookSquare
+                    <span className="icon-footer">
+                        <a
+                            href="https://www.facebook.com/TheHubAUBG"
+                            target="_blank"
                             style={{ backgroundColor: iconcolor }}
```

Comment:
I don't remember what was the idea of this style property. Is this for the background or the icons themselves? Probably we should have two properties, one for the background and one for the icon's actual Color. Furthermore, can you rename the iconcolor prop to iconColor? We should stick to camelCase.